SCons/Tool/MSCommon/vc.py

```diff
@@ -1020,6 +1055,279 @@ def script_env(script, args=None):
 
     return cache_data
 
+def _msvc_notfound_policy_lookup(symbol):
+
+    try:
+        notfound_policy_def = _MSVC_NOTFOUND_POLICY_EXTERNAL[symbol]
+    except KeyError:
+        err_msg = "Value specified for MSVC_NOTFOUND_POLICY is not supported: {}.\n" \
+                  "  Valid values are: {}".format(
+                     repr(symbol),
```

Comment:
why `repr()`? Aren't these just strings already?

Reply:
> why repr()? Aren't these just strings already? (applies to further instances too)

When using repr, the strings will be quoted:
```
Code:
    print("value: {}".format('x'))
    print("value; {}".format(repr('x'))
Output:
    values: x
    values: 'x'
```
My usual preference is that string values are quoted in messages.  Possible distinction when the string contents could be viable other types (e.g., 5 vs '5').

At the end of the day, your choice.

Reply:
> > why repr()? Aren't these just strings already? (applies to further instances too)
> 
> When using repr, the strings will be quoted:

This is actually a good point.  There are lots of places where SCons manually adds quote marks instead - mainly tests. So we could potentially chang this usage: 
```py
test.skip_test("skipping as test on non-x86 platform '%s'\n" % sys.platform)
```
To
test.skip_test(f"skipping as test on non-x86 platform {sys.platform!r}")
```
I don't know if this ends up actually being less reabable due to the inline repr notation (and sorry for hijacking a part of this thread to ask a question about usage elsewhere).  @bdbaddog have any opinions?

Reply:
I'd say yes, though as @mwichmann pointed out on discord, this pattern is used elsewhere in the SCons codebase.